stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -102,10 +102,8 @@ lazy_static! {
         format!("{}\n{}", BOOT_CODE_POX_MAINNET_CONSTS, POX_3_BODY);
     pub static ref POX_3_TESTNET_CODE: String =
         format!("{}\n{}", BOOT_CODE_POX_TESTNET_CONSTS, POX_3_BODY);
-    pub static ref POX_4_MAINNET_CODE: String =
-        format!("{}\n{}", BOOT_CODE_POX_MAINNET_CONSTS, POX_4_BODY);
-    pub static ref POX_4_TESTNET_CODE: String =
-        format!("{}\n{}", BOOT_CODE_POX_TESTNET_CONSTS, POX_4_BODY);
+    pub static ref POX_4_MAINNET_CODE: String = format!("{}", POX_4_BODY);
+    pub static ref POX_4_TESTNET_CODE: String = format!("{}", POX_4_BODY);
```

Comment:
It might make more sense to just have `POX_4_CODE`, and remove these two variables since they're now the same thing.

Reply:
done